app/views/admin/users/index.html.erb

```diff
@@ -3,71 +3,80 @@
 </div>
 
 <%= render 'shared/paginate_and_filter_by_district',
-           form_url: admin_users_path,
-           current_admin: current_admin,
-           district: @district,
-           per_page: @per_page,
-           scope_namespace: [:manage, :user, Facility] %>
+    form_url: admin_users_path,
+    current_admin: current_admin,
+    district: @district,
+    per_page: @per_page,
+    scope_namespace: [:manage, :user, Facility] %>
 
-<% @users.group_by(&:facility).each do |facility, users| %>
-  <h2 class="mt-3"  id="facility_name"><%= facility.name %></h2>
-  <% users.sort_by(&:full_name).each do |user| %>
-    <div class="card" name="<%= facility.name %>">
-      <div class="row" name="<%= user.full_name %>">
-        <div class="col-md-7">
-          <h5 class="mb-1">
-            <%= link_to user.full_name, [:admin, user] %>
-            <% unless user.sync_approval_status == 'allowed' %>
-              <span class="ml-2 badge badge-pill badge-primary <%= user.sync_approval_status %>"><%= user.sync_approval_status&.capitalize %></span>
-            <% end %>
-          </h5>
-          <div>
-            <% unless user.sync_approval_status == 'allowed' %>
-              <%= user.sync_approval_status_reason %>.
-            <% end %>
+<% if @users.exists? %>
```

Comment:
[how-to-check-if-a-record-exists](https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html)